themes/default/content/docs/concepts/how-pulumi-works.md

```diff
@@ -15,7 +15,6 @@ aliases:
 - /docs/tour/programs-resources/
 - /docs/tour/programs-stacks/
 - /docs/intro/concepts/how-pulumi-works/
-- /docs/intro/concepts/programming-model/
```

Comment:
Removing this line prevents the How Pulumi Works page from "squatting" the `/docs/intro/concepts/programming-model` URL.